tests/services/kuksaService.test.js

```diff
@@ -0,0 +1,117 @@
+const models = require('../../domain/models')
+const kuksaService = require('../../services/kuksaService')
+require('../handleTestDatabase')
+
+var tosuEventsWithWrongTitle
+var tosuEvents
+var kuksaEvents
+var scout
+
+beforeEach(async () => {
+  scout = await models.Scout.create()
+  const kuksaId1 = Math.floor(Math.random() * 1001)
+  const kuksaId2 = Math.floor(Math.random() * 1001)
+
+  // Already parsed by parseKuksaEvents():
+  kuksaEvents = [
+    { id: "kuksa" + kuksaId1, title: "kuksa eventti 1", kuksaEvent: true, kuksaEventId: kuksaId1 },
+    { id: "kuksa" + kuksaId2, title: "kuksa eventti 2", kuksaEvent: true, kuksaEventId: kuksaId2 },
+  ]
+
+  tosuEventsWithWrongTitle = []
+  tosuEventsWithWrongTitle.push(await models.Event.create({title:"test 1", scoutId: scout.id, kuksaEventId: kuksaId1}))
+  tosuEventsWithWrongTitle.push(await models.Event.create({title:"test 2", scoutId: scout.id}))
+  tosuEventsWithWrongTitle.push(await models.Event.create({title:"test 3", scoutId: scout.id}))
+
+  tosuEvents = []
+  tosuEvents.push(await models.Event.create({title:"kuksa eventti 1", scoutId: scout.id, kuksaEventId: kuksaId1}))
+  tosuEvents.push(await models.Event.create({title:"test 2", scoutId: scout.id}))
+  tosuEvents.push(await models.Event.create({title:"test 3", scoutId: scout.id}))
+
+})
+
+test('Sync title from kuksa event to tosu event, removes kuksa source event', async () => {
+  const events = await kuksaService.syncEvents(kuksaEvents, scout.id)
+  tosuEventsWithWrongTitle[0].dataValues.title = "kuksa eventti 1" // Expect to update "test 1" to this title from kuksa
+  expect(hasEvents(events, tosuEventsWithWrongTitle, kuksaEvents)).toBe(true)
+})
+
+test('Event deleted in Kuksa is deleted in tosu as well', async () => {
```

Comment:
I think this test wouldn't fail even if the element wasn't deleted. (hasEvents only tests if events in events1 and events2 are found once in the group. It allows extra elements in group.)